### ISSUE TITLE
TA comments

### ISSUE THREAD

**Issue by Rashedul** [STAT545-UBC-students/hw02-rasiimwe#4]
- Your readme file had a good description of your directory but did not link the files.

- Your commit messages were well described. You can add chunk names here: https://github.com/STAT545-UBC-students/hw02-rasiimwe/tree/master/hw02_files/figure-markdown_github

- Indentation:

```
#bad
p0 <- df %>% filter(cont=="Asia") %>% 
  ggplot(aes(gdppc))+ geom_histogram(bins=40)+theme_light()+ ggtitle("GDP per capita of Asia for all years")+theme(plot.title = element_text(hjust = 0.5))+ labs(x = "GDP Per Capita - Asia")

#good
p0 <- df %>% filter(cont=="Asia") %>% 
  ggplot(aes(gdppc)) +
  geom_histogram(bins=40) + 
  theme_light() + 
  ggtitle("GDP per capita of Asia for all years") +
  theme(plot.title = element_text(hjust = 0.5)) + 
  labs(x = "GDP Per Capita - Asia")

``` 

- You put a great amount of effort on your hw02. You explored and explained this carefully.

- You explored various plot types nicely, but nothing in the section of But I want to do more!

- Excellent JOB!

**Comment by rasiimwe** [comment]
@Rashedul 

Hi Rashedul,

Thanks for your great comments and advice. I didn't label a section called **"I want to do more"** but you will see in my work that I integrated the use of  `knitr::kable()` almost everywhere. I also used  **more of the dplyr functions for operating on a single table** as advised for students who want to do more.

Please look through my work. You will see these.

Many thanks
Becky

**Comment by Rashedul** [comment]
@rasiimwe,

Can you point me where you addressed this?

"Evaluate this code and describe the result. Presumably the analyst’s intent was to get the data for Rwanda and Afghanistan. Did they succeed? Why or why not? If not, what is the correct way to do this?"

```
filter(gapminder, country == c("Rwanda", "Afghanistan"))
```

**Comment by rasiimwe** [comment]
@Rashedul 

Yes, thanks for pointing that out, I didn't get time to do that small section, but I hope you saw the rest.

Thanks

**Comment by rasiimwe** [comment]
@Rashedul 

Will my grade be changed to reflect the components mentioned from the "I want to do more" section? - apart from one 😄.

**Comment by rasiimwe** [comment]
@Rashedul @vincenzocoia 

Hi,
Kindly review and respond to this issue.

Just to follow up on a few things:
Does naming of chunks take up a considerable grade of a student?

Were 60 points taken by the "I want to do more" question on:

> "Evaluate this code and describe the result. Presumably the analyst’s intent was to get the data for Rwanda and Afghanistan. Did they succeed? Why or why not? If not, what is the correct way to do this?"
> filter(gapminder, country == c("Rwanda", "Afghanistan"))

In most of my work I applied this 👇 style of coding:

```{r}
df %>% 
  ggplot(aes(reorder(cont, gdppc), gdppc)) +
  geom_boxplot(outlier.colour="firebrick",fill = "orange1") + 
  geom_jitter(alpha=0.2)+
  scale_y_log10()+
  labs(y="Mean GDP Per Capita", x="Ordered Continent")
```


Does forgetting to do so on one or two lines take up a considerable grade of a student? 

It would help to be in the know so that in my future assignments I avoid grade draining hitches. It also helps to inform a student what sections let them down and the points they lost in the identified sections given the enormous effort we put in our assignments.

Looking forward to hearing from you
Many thanks

**Comment by Rashedul** [comment]
@rasiimwe,

Thanks for your feedback. For future home works, try to address all the tasks and use consistent coding to get full marks.

It is obvious that you put great amount of effort. As I mentioned before that you excellent job in this home work, I revised your grade.

**Comment by rasiimwe** [comment]
@Rashedul 

Thanks Rashedul, I will do as advised for my future assignments.

Many thanks
Rebecca
